app/controllers/v1/submission_entries_controller.rb

```diff
@@ -16,9 +16,15 @@ def create
 
   def update
     submission_file = SubmissionFile.find(params[:file_id])
-    submission_file.entries.find(params[:id])
+    entry = submission_file.entries.find(params[:id])
+    entry.aasm.current_state = submission_entry_params[:status]
```

Comment:
Have you considered checking if `submission_entry_params[:status]` is empty? If it's omitted, would this set the entry's status to `''` ?

Reply:
Yeah I checked, I think aasm already checks that current_state can't be nil, so I thought checking again will be a bit redundant, but happy to add it.

Reply:
Happy to leave this as-is. Perhaps add a test around updating `validation_errors` and not `status` to verify that `aasm` doesn't blow up in this case?